editor.planx.uk/src/@planx/components/shared/Schema/Fields.tsx

```diff
@@ -17,45 +26,49 @@ import ErrorWrapper from "ui/shared/ErrorWrapper";
 import Input from "ui/shared/Input";
 import InputRowLabel from "ui/shared/InputRowLabel";
 
-import { DESCRIPTION_TEXT, ERROR_MESSAGE } from "../../shared/constants";
-import BasicRadio from "../../shared/Radio/BasicRadio";
-import type {
-  ChecklistField,
-  DateField,
-  NumberField,
-  QuestionField,
-  TextField,
-} from "../model";
-import { useListContext } from "./Context";
+import { DESCRIPTION_TEXT, ERROR_MESSAGE } from "../constants";
+import BasicRadio from "../Radio/BasicRadio";
+
+type Props<T extends Field> = {
+  formik: FormikProps<SchemaData>
+  activeIndex: number;
+} & T;
```

Comment:
Feel free to ignore as this is just my preference on structure, but I would like to see types and helpers in a separate file to leave component files cleaner :)